sympy/geometry/polygon.py

```diff
@@ -374,6 +374,63 @@ def centroid(self):
             cy += v*(y1 + y2)
         return Point(simplify(A*cx), simplify(A*cy))
 
+
+    def second_moment_of_area(self, p='centroid'):
+        """Returns the second moment and product moment of area of a two dimensional polygon.
+
+        Parameters
+        ==========
+
+        p : Point, two-tuple of sympifiable objects, or 'centroid'(default='centroid')
+            p is the point about which second moment of area is to be found.
+            if centroid, it will be calculated about the axis passing through the
```

Comment:
1. If (i.e. capital I)
2. ``p='centroid'``

making it: If \`\`p='centroid'\`\` it will be calculated ...